### PR TITLE
Merge AtomHelper, DydxHelper & KujiraHelper into CosmosHelper

### PR DESCRIPTION
**Pull request by yvebe**

### CODE REVIEW

File: app/src/main/java/com/vultisig/wallet/data/repositories/GasFeeRepository.kt [hunk 1]
```diff
@@ -83,7 +82,7 @@ internal class GasFeeRepositoryImpl @Inject constructor(
             Chain.Dydx -> {
                 val nativeToken = tokenRepository.getNativeToken(chain.id)
                 TokenValue(
-                    value = DydxHelper.DYDX_CHAIN_GAS_LIMIT.toBigInteger(),
+                    value = 2500000000000000L.toBigInteger(),
```
Comment (yevhen1sec) [inline app/src/main/java/com/vultisig/wallet/data/repositories/GasFeeRepository.kt:85]: maybe better create constant?

**Review by johnnyluo — APPROVED** [review]
LGTM

**Comment by johnnyluo** [comment]
Can you please send a cosmso transaction?

**Comment by johnnyluo** [comment]
https://www.mintscan.io/cosmos/tx/628DE1AFF71189419DA20CECFBE6E8D302343D510D4ADAEE3FEF4FA552DD64B1

**Pull request merged**